presto-server-rpm/src/main/resources/dist/config/jvm.config

```diff
@@ -5,5 +5,5 @@
 -XX:+ExplicitGCInvokesConcurrent
 -XX:+HeapDumpOnOutOfMemoryError
 -XX:+UseGCOverheadLimit
--XX:OnOutOfMemoryError=kill -9 %p
+-XX:+ExitOnOutOfMemoryError
```

Comment:
This change prevents the RPM installed version from running, so we should also change `preinstall` to require that version.